pwndbg/aglib/argv.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import pwndbg
+import pwndbg.aglib.arch
+import pwndbg.aglib.memory
+import pwndbg.aglib.regs
+import pwndbg.aglib.typeinfo
+from pwndbg.dbg import EventType
+
+#: Total number of arguments
+argc_numbers: int = None
+
+#: Pointer to argv on the stack
+argv_ptr: int = None
+
+#: Pointer to envp on the stack
+envp_ptr: int = None
+
+#: Total number of environment variables
+envc_numbers: int = None
+
+
+@pwndbg.dbg.event_handler(EventType.START)
+def update() -> None:
```

Comment:
If we can, then we should not do this as an event handler and instead fetch those values from stack memory only if argv/envp etc is used.

I mean: most of the time the user does not need those at all?